dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:14-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Copy the package.json and package-lock.json files to the working directory
+COPY package*.json ./
+
+# Install app dependencies
+RUN npm install
+
+# Copy the rest of the app files to the working directory
+COPY . .
+
+# Expose port 3000
+EXPOSE 3000
```

Comment:
I made the same mistake in my Dockerfile, but the `index.js` is using an environment variable `PORT` to set the port, or defaulting to 1333. So this should be 1333 or a PORT environment variable should be set in the Dockerfile.

Reply:
Yes, lets set a PORT env var please